src/dmd/cli.d

```diff
@@ -182,11 +182,12 @@ struct Usage
         Option("color",
             "turn colored console output on"
         ),
-        Option("color=[on|off]",
+        Option("color=[auto|on|off]",
```

Comment:
I suggest using `default` instead of `auto`

Reply:
`auto` is what existing tools use (see e.g. `man diff` or `man git-diff`) (though they also use `always`/`never` instead of `on`/`off`).

Reply:
Maybe change `on`/`off` to `always`/`never` to match, and leave `on`/`off` as undocumented synonyms?

Reply:
I like on/off. It's unambiguous and short. Also, when I first read this text, I wondered what the default setting was. I had to go read the implementation. By using `default` it's one less thing the user has to figure out.

Reply:
`auto` does not imply `default`.

Reply:
Using "default" just shifts the problem: now, the question becomes "OK, but what does the default do? Is it on or off?" 

The help text (or manual page, if there's no room in the help text) should just mention what the default is.

Reply:
> what does the default do?

The same documentation as for `auto`:
```
$(LI $(B default): use colored output if a tty is detected)
```

Reply:
There was enough room to add a clarification of what the default is/does, so I added one.